modules/dataquery/test/TestPlan.md

```diff
@@ -1,119 +1,201 @@
+
+13. Assert that clicking some selected fields in the main table (greyed-out lines in the table) also remove them from the selected column (toggle interaction).
+
+## Other Parameters
+
+Select the `Other parameters` field category in the top dropdown bar.
+
+1. Assert that default visits fields for this category are loaded.
+
+2. Assert that clicking on a field such as `VisitLabel` or `Project` greys the line **and** adds visit information on the same line.
+
+3. Assert that removing visits from `Default Visits` allows to reselect them, either by clicking the right arrow, or by entering corresponding text in the text area.
+
+## Sync not sync
+
+[ ] uncheck `Sync with selected fields` and add all visits from `Default Visits` box. Then click some fields in the list so that theyj are greyed out.
+
+1. Assert that selected fields each have a visit selection box.
+
+2. Assert that all visits displayed in the field lines are the same as those in the `Default Visits` box.
+
+3. Assert that all visits are described in the selected fields in the right column.
+
+Remove some visits from `Default Visits` box and click on `Other Fields`.
+
+4. Assert that new field visits only reflects what the `Default Visits` box is showing.
+
+5. Assert that the newly selected fields in the right column have the new default visits.
+
+6. Check `Sync with selected field` checkbox and assert that all fields match the default.
+
+7. Make sure the `Selected Fields` are still updateable by ex-ing out visits in the main table.
+
+## Filters
+
+Click on `Add Filters` in `Next Steps` and make sure no filter is already selected.
+
+1. Assert that the sentence `Currently querying for ALL candidates` should be displayed under the light-blue notification.
+
+2. Assert the light-blue notification saying `Note that only candidates which you have permission to access in LORIS are included in results. Number of results may vary from other users running the same query' is there`.
+
+3. Assert that a preview of the number of candidates matched is displayed in the top-right hand corner.
+
+4. Make sure that the `Add Condition` button triggers field select modal.
+
+Add a field as a filter
+
+5. Assert that the condition now appears in the filter list.
+
+Select your filter, and enter your condition, and click `Submit`. 
+
+6. Assert that: `Add Condition` and `Import from CSV` buttons are now replaced by `Add "and" condition` and `Add "or" condition`.
+
+7. Assert that the top-right corner is updated.
+
+8. Remove the added conditions with the trashbin icon on the right.
+
+9. Assert that `Import from CSV` button features an upload modal.
+
+10. Assert that: sending something different than CSV ends with an `Invalid <Something>` alert.
+
+11. Test file import by creating and importing different files with the preset options (candidates vs. sessions, DCCID vs. PSCID, with or without headers).
```

Comment:
Elsewhere in this doc the instructions are not numbered, but the assertions are. Removing the number here would be more consistent with the rest of it
```suggestion
6. Assert that: `Add Condition` and `Import from CSV` buttons are now replaced by `Add "and" condition` and `Add "or" condition`.

7. Assert that the top-right corner is updated.

Remove the added conditions with the trashbin icon on the right.

8. Assert that `Import from CSV` button features an upload modal.

9. Assert that: sending something different than CSV ends with an `Invalid <Something>` alert.

10. Test file import by creating and importing different files with the preset options (candidates vs. sessions, DCCID vs. PSCID, with or without headers).
```